### PR TITLE
Enforce CSV names style for `pytest.mark.parametrize`

### PR DESCRIPTION
**Pull request by akx**
As mentioned by @radarhere in 
https://github.com/akx/Pillow/pull/7, Pillow's `pytest.mark.parametrize` house style is `csv` – this configures Ruff to enforce that and fixes the stray tuples.

* Ruff rule reference: https://docs.astral.sh/ruff/rules/pytest-parametrize-names-wrong-type/

### CODE REVIEW

File: Tests/test_file_dds.py [hunk 1]
```diff
@@ -152,7 +152,7 @@ def test_sanity_ati2_bc5u(image_path: str) -> None:
 
 
 @pytest.mark.parametrize(
-    ("image_path", "expected_path"),
+    "image_path,expected_path",
```
Comment (hugovk) [inline Tests/test_file_dds.py:155]: The suggestion in https://github.com/akx/Pillow/pull/7 is with spaces after the comma:

> * Replaced `("lut_mode", "table_size")` with `"lut_mode, table_size"`. It seems simpler to me, and is [the style that Pillow is using elsewhere](https://github.com/python-pillow/Pillow/blob/11b4df3ff9c02757eaf687ac13374f3525c8cb71/Tests/test_file_tiff_metadata.py#L195)

Can the formatter do it like that too?

```suggestion
    "image_path, expected_path",
```

Comment (akx) [reply Tests/test_file_dds.py]: The autofix would not (did not) add a space, but the linter check is fine with strings that have or don't have spaces.

Re-pushed with spaces :)

**Review by radarhere — APPROVED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -129,6 +130,8 @@ lint.per-file-ignores."Tests/oss-fuzz/fuzz_font.py" = [
 lint.per-file-ignores."Tests/oss-fuzz/fuzz_pillow.py" = [
   "I002",
 ]
+lint.flake8-pytest-style.parametrize-names-type = "csv"
+
 lint.isort.known-first-party = [
```
Comment (radarhere) [inline pyproject.toml:135]:
```suggestion
lint.flake8-pytest-style.parametrize-names-type = "csv"
lint.isort.known-first-party = [
```
Nitpick

Comment (akx) [reply pyproject.toml:135]: A linter forced the order I committed.

Comment (radarhere) [reply pyproject.toml:135]: I wasn't suggesting the order be changed, simply removing the blank line in the middle.

**Pull request merged**